### DISCUSSION TITLE
self join in the style of Pandas merge() ...

### DISCUSSION THREAD

**Discussion by gjvc** [jtablesaw/tablesaw#1160] [Q&A]
In Pandas, I have this .merge() operation, which is actually a self-join (merging a dataframe df with itself)

`        df = pd.merge( left=rx, right=tx, how='outer', left_on='process_ts', right_on='trigger_ts', suffixes=('_rx', '_tx'), indicator=True )
`

How may I go about this in tablesaw?

**Comment by lwhite1** [comment]
Look at the syntax for joins. The details are in the join package. It
doesn't do all that but it's pretty close. You will have to rename the
suffixes yourself after the join completes.
